web/src/screens/roleShow.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
 import {Container, Col, Row} from 'react-bootstrap'
-import {jobShowHeader as Header, jobShowCarousel as JobGraphics, jobShowSummary as Summary } from '../components/Cards'
+import {JobShowHeader as Header, JobShowCarousel as JobGraphics, JobShowSummary as Summary } from '../components/Cards'
```

Comment:
If your components need different names, rename the actual components, not the imports